docs/ballerina-by-example/examples/grpc-bidirectional-streaming/grpc-bidirectional-streaming.bal

```diff
@@ -14,7 +14,7 @@ endpoint grpc:Service ep {
     serverStreaming:true,
     generateClientConnector:false}
 service<grpc:Endpoint> chat bind ep {
-    map consMap = {};
+    map consMap = new;
```

Comment:
Ideally this should work without initialization. Please check. @sameera-jayasoma says new is not supposed to even work with map. @lankavitharana @hasithaa et al please check